source/conf.py

```diff
@@ -106,6 +106,9 @@
 # a list of builtin themes.
 #html_theme = 'default'
 import sphinx_rtd_theme
+extensions = [
+    "sphinx_rtd_theme",
+]
```

Comment:
Please remove those three lines. Otherwise, sphinx won't work. Besides, I have put this in Line 36~